### ISSUE TITLE
It is recommended to upload the software to Microsoft store

### ISSUE THREAD

**Issue by BraveMomo** [BabylonJS/Editor#476]
It is recommended to upload the software to the Microsoft store, which is also convenient for users to update and can attract more users

**Comment by BraveMomo** [comment]
Of course, the download address of the web version still needs to be retained

**Comment by julien-moreau** [comment]
Hey @BraveMomo !
Thanks for the feedback. It is in the backlog including the AppStore.
